frontend/catalyst/jax_primitives.py

```diff
@@ -518,8 +520,8 @@ def _apply_registered_pass_lowering(
     # jax_ctx.module_context.module holds a reference to @inner
     #
     # This means that it's parent is @root.
-    parent_module = module.parent
-    for op in reversed(parent_module.regions[0].blocks[0].operations):
+    # parent_module = module.parent
+    for op in reversed(module.regions[0].blocks[0].operations):
```

Comment:
Should the "readme" illustration above be updated too?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1323/commits/4ab3ddc8b0c19057ca1a02ffc365204d61e3b325